lib/Commander.js

```diff
@@ -58,6 +59,83 @@ module.exports = (function() {
 
 
 
+
+
+
+
+
+
+
+
+    Commander.prototype.checkFourEyes = function() {
+        var commander = this;
+        var organization = null;
+        var repo = null;
+
+        this.askForOrg()
+        .then(function (_organization) {
+            organization = _organization;
+            return commander.askForRepo (organization);
+        })
+        .then(function (_repo) {
+            repo = _repo;
+            console.log ('Checking repo ' + organization + '/' + repo + '.');
+            return commander.githubWrapper.getPullRequests (organization, repo);
+        })
+        .then(function (data) {
+            return Q.allSettled (data.map (function (pullRequest) {
+                return commander.githubWrapper.getPullRequest (organization, repo, pullRequest.number);
+            }));
+        })
+        .then(function (resolves) {
+
+            var content = '';
+
+            for (var i = 0; i < resolves.length; i++) {
+                var pr = resolves [i];
+
+                if (resolves [i].state == 'fulfilled') {
```

Comment:
Maybe return `Q.reject(resolves[i].reason)` if state is not fulfilled. So we can see the error.